src/features/landing/components/AboutVilnaSection/AboutVilnaSecondaryImages/AboutVilnaSecondaryImages.tsx

```diff
@@ -0,0 +1,145 @@
+import { Box, Container } from '@mui/material';
+
+import AboutVilnaSecondaryShape from '../../../../../../public/assets/img/AboutVilnaSecondaryShape.png';
```

Comment:
this is difficult to read, read this and implement https://medium.com/thefork/escaping-relative-import-paths-hell-in-javascript-9f258baeb15e